tests/_test_mscolab/test_server.py

```diff
@@ -130,14 +132,45 @@ def test_get_user(self):
     def test_delete_user(self):
         assert add_user(self.userdata[0], self.userdata[1], self.userdata[2])
         with self.app.test_client() as test_client:
+            # Case 1 : The user has no profile image set
             token = self._get_token(test_client, self.userdata)
             response = test_client.post('/delete_own_account', data={"token": token})
             assert response.status_code == 200
-            data = json.loads(response.data.decode('utf-8'))
-            assert data["success"] is True
-            response = test_client.post('/delete_own_account', data={"token": "dsdsds"})
+            assert response.get_json()["success"] is True
+            # ToDo: Check if user token was cleared after deleting account as assert returns True instead of False
+            # assert verify_user_token(config_loader(dataset="mscolab_server_url"), token) is False
+
+            # Case 2 : The user has a custom profile image set
```

Comment:
Shouldn't this test case also check that the deleted users profile image is removed?